src/ripple/rpc/handlers/Tx.cpp

```diff
@@ -53,12 +52,14 @@ struct TxResult
     Transaction::pointer txn;
     std::variant<std::shared_ptr<TxMeta>, Blob> meta;
     bool validated = false;
+    std::optional<std::string> ctid;
```

Comment:
I'm not following why this is a string instead of a u64. (I'm not saying there isn't a reason, I'm just not seeing why it's done).

Reply:
That makes sense. Thanks @RichardAH!